pkg/config/config.go

```diff
@@ -9,16 +9,17 @@ import (
 )
 
 const (
-	FALCO_EBPF_ENGINE  = "falco"
-	CILIUM_EBPF_ENGINE = "cilium"
+	EBPFEngineFalco  = "falco"
+	EBPFEngineCilium = "cilium"
+	CONFIG_ENV_VAR   = "CONFIG_ENV_VAR"
 )
 
 type Config struct {
 	data ConfigDataInterface
 }
 
 func (cfg *Config) getConfigFilePath() (string, bool) {
-	return os.LookupEnv("SNIFFER_CONFIG")
+	return os.LookupEnv(CONFIG_ENV_VAR)
```

Comment:
are you sure it was rebased correctly? I think this was fixed in your previous PR?